include/certifier_framework.h

```diff
@@ -373,7 +374,7 @@ class secure_authenticated_channel {
 
 bool server_dispatch(const string &host_name,
                      int           port,
-                     string &      asn1_root_cert,
+                     const string &asn1_root_cert,
```

Comment:
Same reason here, too ... Input `string` is never changed.

Reply:
yup